hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/Rollout.java

```diff
@@ -146,12 +146,14 @@
         /**
          * Rollout could not be created due to errors, might be a database
          * problem during asynchronous creating.
+         * @deprecated legacy status is not used anymore
```

Comment:
I don't understand, which field you mean?
Only those two enum instances are deprecated and not used anymore for any new Rollouts, but can still appear in older Rollouts.